app/cdap/components/GridTable/components/GridHeaderCell/index.tsx

```diff
@@ -20,21 +20,89 @@ import TypographyComponent from '../Typography';
 import { useGridHeaderCellStyles } from './styles';
 import { IGridHeaderCellProps } from './types';
 
+const headerSelectedIcon = (
```

Comment:
1. What does this look like?
2. Please move it to a separate file

Reply:
Moved it to a separate file .